data-processing/entities/citations/commands/resolve_bibitems.py

```diff
@@ -52,7 +52,7 @@ def load(self) -> Iterator[MatchTask]:
                     references_path,
                     arxiv_id,
                 )
-                return
+                continue
```

Comment:
Making sure I understand the changes here (the return -> continues)... Before, we would stop yielding MatchTasks as soon as we found an id where we found that it didn't have references or bibitems (not exactly sure what they mean in this context, though a guess is that references are the things we got from the S2 API and bibitems are the things scholarphi pulled from the paper through its pipeline?). But this means we may have lost MatchTasks for other ids after it that do have this information. These changes fix that in when we don't find references or bibitems for an id, we go onto the next id rather than returning immediately. Is that right?

Reply:
> references are the things we got from the S2 API and bibitems are the things scholarphi pulled from the paper through its pipeline

Yes, that's correct.

I think it's a bug to have `return` instead of `continue`. Though it doesn't matter much now because we've only been processing one paper at a time (through the flag `--one-paper-at-a-time`).